src/Uno.Extensions.Reactive.Generator/Bindables/BindableViewModelGenerator.cs

```diff
@@ -32,13 +34,15 @@ where IsSupported(type)
 
 		foreach (var vm in viewModels)
 		{
-			yield return (vm, Generate(vm));
+			yield return (vm.ToString(), Generate(vm));
```

Comment:
Should this be the Name property? I think it'd be better to be explicit about which property is being used here, rather than using ToString and assuming it's returning a specific property value

Reply:
I usually agree, but for the `INamedTypeSymblol` the `ToString()` is documented to return the fullname in a readable way (i.e. `<>` instead of ` ``1 `) and it's the common/right way to deal with it in code gen. The `Name` is only the class name, which could conflict with classes in different namespaces.